cmd/main.go

```diff
@@ -76,7 +76,11 @@ func main() {
     var localOnly = flag.Bool("local", false,
         "If set, it only listens to incoming requests from the local host")
     var port = flag.Int("http-port", 7399, "HTTP Server port for the REST API")
-    var redisUrl = flag.String("redis", "", "URI for the Redis cluster (host:port)")
+    var redisUrl = flag.String("redis", "", "For single node redis instances: URI "+
+        "for the Redis instance (host:port). For redis clusters: a comma-separated list of redis nodes. "+
+        "If using an ElastiCache redis cluster with cluster mode enabled, you can supply the configuration endpoint.")
```

Comment:
```suggestion
        "If using an ElastiCache Redis cluster with cluster mode enabled, you can supply the configuration endpoint.")
```
Does it apply on any Redis deployment in cluster mode? or just Elasticache? If any, then we can remove the "Elasticache" part

Reply:
Sure, updated